apps/system/js/browser_context_menu.js

```diff
@@ -207,8 +215,8 @@
             label: _('open-in-new-private-window'),
             callback: this.openUrl.bind(this, uri, true)
           }, {
-            id: 'bookmark-link',
-            label: _('add-link-to-home-screen'),
```

Comment:
Unfortunately I don't think we can safely remove this until we switch to the new homescreen, because pinned pages won't appear anywhere?